src/schema/books-model.js

```diff
@@ -49,8 +48,6 @@ function addBook(book) {
 function addCharacter(character) {
     return db('characters')
         .insert(character, 'id')
```

Comment:
Deleting the `.then` means you aren't waiting for the promise to resolve before you return, so you can do an `async/await` or do the `.then` where you call this to make sure it happened.

Reply:
So if I just want to return the character I've just updated, can I use
`.then(character)`
?